packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DatePickerInput.tsx

```diff
@@ -34,6 +39,7 @@ const DatePickerInput = (props: DatePickerInputProps) => {
     formatDate,
     isFromOrTo,
     selectedDate,
+    setFocus = false,
```

Comment:
cant we forward the ref instead?

Reply:
yeah, i'm ok with keeping all the setFocus stuff in `-Filter` components for now.

Reply:
Moved to DateRangeFilter